services/ingest-update-center

```diff
@@ -29,15 +29,33 @@ let ingest = {
         signature: '4faefcc46299557efaa1c896835a2413ad0646201e40938b4207dbd4cfecf33303cf5f28fd1f53cb71146c45d12c7fa34d2d87d312acf9fd7964c08506d2278d',
       },
     },
-     {
-       groupId: 'io.jenkins.plugins',
-       artifactId: 'essentials',
-       url: 'http://repo.jenkins-ci.org/incrementals/io/jenkins/plugins/essentials/0.3-rc20.a3f76cdd1348/essentials-0.3-rc20.a3f76cdd1348.hpi',
-       checksum: {
-         type: 'sha512',
-         signature: 'e1cdf271eabed428c2198c4f7a49c5dea7fc8850987bcf63b87b74936a3cc426896a4b7c9c974f0fe3abd61071ea5a2238538af9f9266f2d37696856809d697a',
-       },
-     },
+    {
+      groupId: 'io.jenkins.plugins',
+      artifactId: 'aws-global-configuration',
+      url: 'https://repo.jenkins-ci.org/incrementals/io/jenkins/plugins/aws-global-configuration/1.0-rc20.ffbe7e5de068/aws-global-configuration-1.0-rc20.ffbe7e5de068.hpi',
+      checksum: {
+        type: 'sha512',
+        signature: 'c78067f7f96668aa9798b7fc735b77dfaa208f4499238d42079ba6a4c9eabbc43adf6b58b9e5cb1dd143b17659f33d23945cda86d09423e17c4b9dad682c5cef',
+      },
+    },
+    {
+      groupId: 'io.jenkins.plugins',
+      artifactId: 'artifact-manager-s3',
+      url: 'https://repo.jenkins-ci.org/incrementals/io/jenkins/plugins/artifact-manager-s3/1.1-rc77.15d7dd937b42/artifact-manager-s3-1.1-rc77.15d7dd937b42.hpi',
+      checksum: {
+        type: 'sha512',
+        signature: '547b6a605aae7585f1c4d0995618735942ce3936bf4560bd11717b7c0c08876d30933a755a596f5be6e7d0834b2e3587fa55913fe3233a6d28857976435aa18d',
+      },
+    },
+    {
+      groupId: 'io.jenkins.plugins',
+      artifactId: 'essentials',
+      url: 'http://repo.jenkins-ci.org/incrementals/io/jenkins/plugins/essentials/0.3-rc20.a3f76cdd1348/essentials-0.3-rc20.a3f76cdd1348.hpi',
```

Comment:
`http` or `https` preferred? Inconsistent between `essentials` and `configuration-as-code`.